go/runtime/bundle/manager.go

```diff
@@ -168,7 +177,7 @@ func (m *Manager) run(ctx context.Context) {
 	}
 
 	// Remove unneeded bundles and update the manifest map accordingly.
-	manifests, err = m.cleanupBundles(manifests, exploded)
+	manifests, err = m.cleanBundlesOnStartup(manifests, exploded)
```

Comment:
```suggestion
	manifests, err = m.cleanOnStartup(manifests, exploded)
```
To be inline with `m.clean()` below.